### PR TITLE
Add PHP 7.2 Support

### PR DESCRIPTION
**Pull request by AndroidKitKat**
Should work. Let me know what I should test if anything

### CODE REVIEW

File: scripts/run.py [hunk 1]
```diff
@@ -102,6 +102,11 @@ def usage(exit_code=0):
         './{executable}',
         ('.rs',)
     ),
+    Language('PHP 7.2',
```
Comment (pbui) [inline scripts/run.py]: Fix indentation

File: tests/test_code_echo.sh [hunk 2]
```diff
@@ -293,3 +293,16 @@ echo
 echo -n "Testing C (Wrong Formatting)... "
 curl -F source=@$SOURCE localhost:9206/code/test-echo
 rm -f $SOURCE
+
+# PHP 7.2
+SOURCE=$(mktmp -t dredd_XXXXXXX.php)
```
Comment (pbui) [inline tests/test_code_echo.sh]: Should be mktemp

Move test to after Rust.

**Comment by AndroidKitKat** [comment]
Fixed as requested.

File: tests/test_code_echo.sh (hunk 2 above)
Comment (AndroidKitKat) [reply tests/test_code_echo.sh]: Fixed

File: scripts/run.py (hunk 1 above)
Comment (AndroidKitKat) [reply scripts/run.py]: Fixed

**Comment by pbui** [comment]
We don't need this for now.

**Pull request closed without merging**